spec/requests/payments/new_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GET /payments/new', type: :request do
+  subject { response }
+
+  context 'without logged user' do
+    before do
+      stub_request(:post, 'https://api.iugu.com/v1/customers')
+        .to_return(status: 200, body: '', headers: {})
+      get '/payments/new'
+    end
+
+    it { is_expected.to have_http_status(:found) }
```

Comment:
Can't you use a `redirect_to` expectation here? It would make the intention more explicit.